fe/fe-core/src/main/java/org/apache/doris/cloud/load/CloudLoadManager.java

```diff
@@ -46,5 +83,189 @@ public long createLoadJobFromStmt(InsertStmt stmt) throws DdlException {
         return super.createLoadJobFromStmt(stmt);
     }
 
+    public LoadJob createLoadJobFromStmt(CopyStmt stmt) throws DdlException {
+        Database database = super.checkDb(stmt.getDbName());
+        long dbId = database.getId();
+        BrokerLoadJob loadJob = null;
+        if (Config.isCloudMode()) {
```

Comment:
line 90 ~ 98 make simple, see line 74;

What's more CloudXX only call when Config.isCloudMode = true. So directory cloud/xxx should no appear Config.isCloudMode